.github/workflows/icr-image-build-and-push.yml

```diff
@@ -38,3 +40,8 @@ jobs:
         uses: ./.github/actions/icr-build-and-push-images
         with:
           tag: ${{ github.event.inputs.tag }}
+      - name: Build and push on release
+        if : github.event_name == 'release' && github.event.action == 'published'
+        uses: ./.github/actions/icr-build-and-push-images
+        with:
+          tag: ${{ github.event.release.tag_name }}
```

Comment:
The main problem with this approach is that now our docker images instead to be 0.2.1 will be v0.2.1. Not a problem at all but it's a change. The only option that I found to fix it it was this one:

https://stackoverflow.com/questions/71630227/github-action-how-to-remove-starting-v-from-variable

But I didn't like at all. What do you think?

Reply:
It is used in the helm packaging action here: https://github.com/Qiskit-Extensions/quantum-serverless/blob/9d72667d9e853c561c647907838a7169abd79cc6/.github/workflows/publish-helm.yaml#L23C51-L23C64

Reply:
I don't have a strong preference either way... it's probably better to leave the tags as is rather than introduce a change (though if we _were_ going to make a change, now is probably the time to do it), but it also could be nice if the image tags matched exactly to the git tags... 

If it's just the syntax that's bugging you, there's always  `sed`  :smile:

Reply:
The helm version usually doesn’t have the “v” prefix.  The GitHub tag usually has the “v” prefix. The docker tag is free format :-).

Reply:
Yeah, I think we can find different formats in different platforms. I will change the code to use it, thanks team 😄